src/gwt/src/org/rstudio/studio/client/common/StudioClientCommonConstants_fr.properties

```
@@ -213,3 +213,4 @@ onProgressAddingLabel=Ajouter un dictionnaire...
 removeDictionaryCaption=Confirmer la suppression
 removeDictionaryMessage=Êtes-vous sûr de vouloir supprimer le dictionnaire personnalisé {0}?
 progressRemoveIndicator=Supprimer le dictionnaire...
+consoleBufferedMessage=\n\n[Débordement de sortie détecté; mettre en mémoire tampon les {0} prochaines lignes de sortie]\n\n
```

```suggestion
consoleBufferedMessage=\n\n[Débordement de sortie détecté; mise en mémoire tampon des {0} prochaines lignes de sortie]\n\n
```

`Débordement de sortie` still feels odd, but I don't know how I would say it.